### PR TITLE
fix: Add 'Items to be pick-up' title above all pickup consignments

### PR DESCRIPTION
**Pull request by JedidiahAwuku**

### CODE REVIEW

**Review by cawmichae — DISMISSED** [review]

File: feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html [hunk 1]
```diff
@@ -13,6 +13,9 @@
     </ng-container>
 
     <!-- pickup consigned entries -->
+    <div class="cx-pickup-order-consigned-entries-header">
```
Comment (WeizhengSap) [inline feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html]: only display the title when there is pickup items

**Review by WeizhengSap — CHANGES_REQUESTED** [review]
check my comment

File: feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html (hunk 1 above)
Comment (JedidiahAwuku) [reply feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html]: Done. Thank you

File: feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html [hunk 2]
```diff
@@ -13,6 +13,14 @@
     </ng-container>
 
     <!-- pickup consigned entries -->
+
+    <div
+      *ngIf="pickupConsignments.length > 0"
```
Comment (WeizhengSap) [inline feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html]: pickupConsignments can be undefined. So, either `pickupConsignments && pickupConsignments.length >0` or `pickupConsignments?.length`

File: feature-libs/order/styles/components/order-details/_order-details-items.scss [hunk 3]
```diff
@@ -34,4 +34,12 @@
     margin: 0.5rem 0 0;
     padding: 0 30px;
   }
+
+  .cx-pickup-order-consigned-entries-header {
+    background-color: var(--cx-color-background);
+    font-weight: 600;
```
Comment (morganm58) [inline feature-libs/order/styles/components/order-details/_order-details-items.scss]: We try to use variables as much as possible. 
This should be ` font-weight: var(--cx-font-weight-semi);`

File: feature-libs/order/styles/components/order-details/_order-details-items.scss [hunk 4]
```diff
@@ -34,4 +34,12 @@
...
+    font-size: 18px;
```
Comment (morganm58) [inline feature-libs/order/styles/components/order-details/_order-details-items.scss]: this should be `font-size: var(--cx-font-size, 1.125rem);`

File: feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html (hunk 2 above)
Comment (JedidiahAwuku) [reply feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html]: Done. Thank you

File: feature-libs/order/styles/components/order-details/_order-details-items.scss (hunk 3 above)
Comment (JedidiahAwuku) [reply feature-libs/order/styles/components/order-details/_order-details-items.scss]: Changed. thank you

File: feature-libs/order/styles/components/order-details/_order-details-items.scss (hunk 4 above)
Comment (JedidiahAwuku) [reply feature-libs/order/styles/components/order-details/_order-details-items.scss]: Changed. thanks

File: feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html [hunk 5]
```diff
@@ -13,6 +13,14 @@
     </ng-container>
 
     <!-- pickup consigned entries -->
+
+    <div
+      *ngIf="pickupConsignments?.length > 0"
```
Comment (WeizhengSap) [inline feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html]: no, either `pickupConsignments?.length` or `pickupConsignments && pickupConsignments.length > 0`.

Comment (JedidiahAwuku) [reply feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html]: I have made the change. Thank you

**Review by morganm58 — APPROVED** [review]

**Comment by JedidiahAwuku** [comment]
Changes within this ticket have been made in [CXSPA-2894](https://jira.tools.sap/browse/CXSPA-2894). Ticket will be closed as no longer required to merge

**Pull request closed without merging**